src/main/java/cn/edu/tsinghua/iotdb/benchmark/workload/SyntheticWorkload.java

```diff
@@ -42,6 +42,7 @@
   private static String[][] workloadValues = initWorkloadValues();
   private static final String CHAR_TABLE =
       "1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+  private static final long timeStampConst = (config.TIMESTAMP_PRECISION.equals("ms")) ? 1L : 100000L;
```

Comment:
If we use "ns" then this time range may be too short to cover the whole time series. However the Long type MAX_VALUE is not large enough when using "ns". A solution may be using string operation although it is a little more complicated.